includes/admin/class-algolia-admin-page-settings.php

```diff
@@ -88,39 +88,39 @@ public function add_settings() {
 
 		add_settings_field(
 			'algolia_application_id',
-			__( 'Application ID', 'algolia' ),
+			esc_html__( 'Application ID', 'algolia' ),
 			array( $this, 'application_id_callback' ),
 			$this->slug,
 			$this->section
 		);
 
 		add_settings_field(
 			'algolia_search_api_key',
-			__( 'Search-only API key', 'algolia' ),
+			esc_html__( 'Search-only API key', 'algolia' ),
 			array( $this, 'search_api_key_callback' ),
 			$this->slug,
 			$this->section
 		);
 
 		add_settings_field(
 			'algolia_api_key',
-			__( 'Admin API key', 'algolia' ),
+			esc_html__( 'Admin API key', 'algolia' ),
 			array( $this, 'api_key_callback' ),
 			$this->slug,
 			$this->section
 		);
 
 		add_settings_field(
 			'algolia_index_name_prefix',
-			__( 'Index name prefix' ),
+span class="pl-s1"> 			esc_html__( 'Index name prefix', 'algolia' ),
```

Comment:
Not sure what happened here.
I used the beta feature of GH to make a suggestion but I guess it went wrong somewhere :/